### PR TITLE
Add pango

### PR DESCRIPTION
**Pull request by ocefpaf**
There is one hiccup with the tests, but the works OK on Linux.

This is needed to get `pyferret` on `OS X`.

Ping @karlmsmith

### CODE REVIEW

**Comment by jakirkham** [comment]
Feel free to pull stuff from PR ( https://github.com/conda-forge/staged-recipes/pull/354 ). I don't think I'll have time to get to this soon.

File: recipes/pango/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        needed, though most of the work on Pango so far has been done in the
+        context of the GTK+ widget toolkit. Pango forms the core of text and font
+        handling for GTK+-2.x.
+    doc_url: http://www.pango.org/
+    dev_url: https://github.com/phuang/pango
+
+extra:
+    recipe-maintainers:
+        - ocefpaf
+        - jakirkham
```
Comment (jakirkham) [inline recipes/pango/meta.yaml]: I'm guessing @ccordoba12 will want to be here too. 😉

**Comment by ocefpaf** [comment]
@ccordoba12 needs this to get the `Qt` recipe going on Linux. So I will skip `OS X` for now to move this forward.

**Comment by ccordoba12** [comment]
Yep, please add me as a maintainer, and merge as soon as this is ready, so
I can restart the Qt build ;-)

**Comment by ocefpaf** [comment]
Done! @ccordoba12 let's get the ``Qt`` re-started :smile:

**Pull request merged**